modules/compute/eks/stack.py

```diff
@@ -319,6 +342,12 @@ def __init__(
                         },
                         "env": {"DISABLE_TCP_EARLY_DEMUX": True},
                     },
+                    "nodeAgent": {
+                        "image": {
+                            "region": self.region,
+                            "account": "602401143452",
```

Comment:
nit - should this be sanitised?

Reply:
Did you mean the account number? This is a standard EKS Control plane team's account from which the images are pulled.

Reply:
Ah sorry thanks